package.json

```diff
@@ -56,13 +69,29 @@
     ]
   },
   "devDependencies": {
+    "@rollup/plugin-alias": "^5.1.1",
+    "@rollup/plugin-commonjs": "^28.0.2",
+    "@rollup/plugin-node-resolve": "^16.0.0",
+    "@rollup/plugin-terser": "^0.4.4",
+    "@rollup/plugin-typescript": "^12.1.2",
+    "@rollup/plugin-url": "^8.0.2",
     "@types/leaflet": "^1.9.16",
     "@types/mocha": "^10.0.10",
     "@types/react": "^19.0.6",
     "@types/react-dom": "^19.0.3",
     "@types/react-grid-layout": "^1.3.5",
+    "postcss-discard-comments": "^7.0.3",
     "prettier": "3.4.2",
+    "rollup-plugin-copy": "^3.5.0",
+    "rollup-plugin-dts": "^6.1.1",
+    "rollup-plugin-peer-deps-external": "^2.2.4",
+    "rollup-plugin-postcss": "^4.0.2",
+    "rollup-plugin-styles": "^4.0.0",
     "typescript": "^5.7.3",
     "web-vitals": "^4.2.4"
+  },
+  "peerDependencies": {
+    "react": "^19.0.0",
```

Comment:
Check the usage of peerDependencies, and remove duplicated ones from the dependencies.